src/main/java/core/basesyntax/MainApp.java

```diff
@@ -3,10 +3,9 @@
 public class MainApp {
     public static void main(String[] args) {
         Machine[] machines = {new Truck(), new Bulldozer(), new Excavator()};
-
-        for (int i = 0; i < machines.length; i++) {
-            machines[i].doWork();
-            machines[i].stopWork();
+        for (Machine machine:machines) {
```

Comment:
The spaces are needed near `:` mark. Please format the code via IDE. And check the result by check style plugin.